src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -80,6 +81,7 @@ public static void init(final IForgeRegistry<Block> registry)
         blockWayPoint = new BlockWaypoint().registerBlock(registry);
         blockHutBarracks = (BlockHutBarracks) new BlockHutBarracks().registerBlock(registry);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Synchronize this lazy initialization of 'blockHutBarracks' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2444)